packages/RNTester/js/examples/TextInput/TextInputExample.js

```diff
@@ -0,0 +1,1615 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const React = require('react');
+
+const {
+  Button,
+  Text,
+  TextInput,
+  View,
+  StyleSheet,
+  Alert,
+  Switch,
+} = require('react-native');
+
+import type {RNTesterExampleModuleItem} from '../../types/RNTesterTypes';
+import {useState, useEffect} from 'react';
+
+const styles = StyleSheet.create({
```

Comment:
Is there any reason styles are declared on top. In every other file, I have seen styles declared in the end.

Reply:
Wasn't a conscious decision, it was the default so I didn't change it.